backend/contentBackend.js

```diff
@@ -1,9 +1,9 @@
 import { get, update, Q, find, remove } from '@reshuffle/db';
-
-import { validateJWT } from './authBackend';
+import { validateUser } from './authBackend';
 import { parseMDLocal } from './parseMD';
 
 const contentPrefix = 'content__';
+// const editedPrefix = 'editedBy__';
```

Comment:
No code in comments, please.  Code (no matter how partial) can always go in git --- you never need to PR it.